examples/inshop/configs/train_inshop.yaml

```diff
@@ -2,9 +2,7 @@ postfix: metric_learning
 
 seed: 42
 precision: 32
-gpus:
-  - 0
-  - 1
+gpus: [0, 1]
```

Comment:
since you change it in only one config, please, rm it

by the way, @DaloroAT will change handling this arguments in the next PR, so you don't need to care about it